backend/docker-compose.yml

```
@@ -0,0 +1,76 @@
+
+services:
+  api:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: uvicorn user_management.main:app --host 0.0.0.0 --port 8000 --reload
+    container_name: api
+    volumes:
+      - .:/app:cached  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)